app/controllers/api/v2/trends_controller.rb

```diff
@@ -0,0 +1,44 @@
+module Api
+  module V2
+    class TrendsController < V2::BaseController
+      include Foreman::Controller::Parameters::Trend
+
+      before_action :find_resource, :only => [:show, :destroy]
+
+      api :GET, "/trends/", N_("List of trends counters")
+      def index
+        @trends = resource_scope_for_index
+      end
+
+      api :GET, "/trends/:id/", N_("Show a trend")
+      param :trend_id, :identifier, :required => true
+      def show
+      end
+
+      api :POST, "/trends/", N_("Create a trend counter")
+      def create
```

Comment:
this and many other actions seems to be missing params definition, see e.g. https://github.com/theforeman/foreman/blob/develop/app/controllers/api/v2/domains_controller.rb#L42-L70 for example how to describe params

Reply:
to reproduced, do this

```
curl -H "Accept:application/json,version=2" -H "Content-Type:application/json" -u admin:changeme -k https://foreman.example.tst/api/v2/trends -X POST -d '{}'
```